libraries/tracking/subscriptions/setup.js

```diff
@@ -32,12 +32,14 @@ export default function setup(subscribe) {
   /**
    * Gets triggered when the app starts.
    */
-  subscribe(appDidStart$, async () => {
+  subscribe(appDidStart$, async ({ getState }) => {
     const clientInformationResponse = await getWebStorageEntry({ name: 'clientInformation' });
 
     const clientInformation = {
       type: get(clientInformationResponse, 'value.device.type', TYPE_PHONE),
       os: get(clientInformationResponse, 'value.device.os.platform', OS_ALL),
+      state: getState(),
```

Comment:
I think, it's not the best idea to merge the `state` and the `subscribe` function to the parameters of `pluginInit()` - see the other pull request.

Reply:
changed the subscribe. But kept the state, because there is no other way